### DISCUSSION TITLE
Why is my clipped page missing .md the extension?

### DISCUSSION THREAD

**Discussion by seyeeL** [coddingtonbear/obsidian-web#31] [Q&A]
![image](https://user-images.githubusercontent.com/71479570/185848455-73164f14-ab7e-4f0f-8af9-888e5dd415cd.png)
link 
https://github.com/coddingtonbear/obsidian-web/issues/27
you can clipper the github issue
![image](https://user-images.githubusercontent.com/71479570/185848643-0719d2b4-d4b2-4dda-a48a-ccf3257b59e4.png)

**Comment by coddingtonbear** [comment]
I'm not able to reproduce a problem from what you've described above; here's what I've done:

- I've created a template that's identical to the one you've posted above (by the way: it'd be easier for folks to help you troubleshoot if you just gave the raw text instead of asking folks to type the text out of a screenshot very carefully).
- I've tried to clip this particular issue (https://github.com/coddingtonbear/obsidian-web/discussions/31).
- I see that a page named "Why is my clipped page missing .md the extension · Discussion #31 · coddingtonbearobsidian-web.md" will be added to my vault.

Something that might help you troubleshoot yourself further is to look at the rendered template shown in the UI when you've selected that template.  You can do that by expanding the "Entry Details" option by clicking on the small arrow shown in this screenshot:

![image](https://user-images.githubusercontent.com/527661/185962624-d825249b-3b9d-4f01-ba07-27c657f67301.png)

If you do that, you can see the contents of the actual API request that will be used for sending content into your notes; in this case:

![image](https://user-images.githubusercontent.com/527661/185962803-f44405c2-8c37-427f-a899-65c2140c71b3.png)

What you'll want to look at is the contents of the "API URL" field to verify that there's a `.md` at the end of the line.
